### PR TITLE
Styling feature

### PR DESCRIPTION
**Pull request by killy10o10**
created a test.MD file and added it to .gitignore. Also, I made a change to the CSS file, using a class selector for my h1 element instead of targeting it directly inside the CSS.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -10,3 +10,52 @@ body {
...
+  height: 300px;
+  margin-left: auto;
+  margin-right: auto;
+  border-radius: 10px;
+  margin-top: 100px;
+}
+
+h1 {
+  color: brown;
+}
```
Comment (thecodechaser) [inline style.css]: - [x] You are using an element selector to style the heading, Please use a class selector instead to follow the project requirements. You can add a class to the heading and use that class in the .css file to style the heading.

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ 

## Hi @killy10o10👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
- No linter errors. ✔️
- GitHub flow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:



## Required Changes ♻️
- [x] Please create a `test.md` file with the text "This file should be ignored by git", and mention this filename `test.md` in `.gitignore` file, and the file will be ignored by git and will not push to Github.
- [x] Please add a description to the pull request.

image
- [x] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 1 above)
Comment (killy10o10) [reply style.css]: Thanks for the review. I'll make the appropriate changes.

**Review by gemmen29 — APPROVED** [review]
Hi @killy10o10, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 



## Optional suggestions

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**